client/Android/Studio/aFreeRDP/build.gradle

```diff
@@ -16,7 +16,7 @@ android {
     signingConfigs {
         release {
             storeFile file(RELEASE_STORE_FILE)
-            storePassword RELEASE_STORE_PASSWORD
+            storePassword RELEASE_KEY_PASSWORD
```

Comment:
Sorry, missed this one, please leave the old name. Will break existing build setups.